Content.Server/Emag/EmagSystem.cs

```diff
@@ -0,0 +1,32 @@
+using Content.Shared.Emag.Components;
+using Content.Shared.Emag.Systems;
+
+namespace Content.Server.Emag
+{
+    public sealed class EmagSystem : EntitySystem
+    {
+        public override void Update(float frameTime)
+        {
+            base.Update(frameTime);
+
+            foreach (var emag in EntityManager.EntityQuery<EmagComponent>())
+            {
+                if (emag.Charges == emag.MaxCharges)
+                {
+                    emag.Accumulator = 0;
+                    continue;
+                }
+
+                emag.Accumulator += frameTime;
+
+                if (emag.Accumulator < emag.RechargeTime)
+                {
+                    continue;
+                }
+
+                emag.Accumulator -= emag.RechargeTime;
+                emag.Charges++;
+            }
```

Comment:
do emags recharge in ss13? i dont really remember this

Reply:
The idea is weaker bursts (so strats like just spending all of your emag charges rushing the armory are worse) for better long term utility. If you think it's too good I'd rather at least try increasing the cooldown or TC cost first before removing it

Reply:
yeah im ok with this being tried at the very least